message-queue-lib/src/cmr/message_queue/test/queue_broker_side_api.clj

```diff
@@ -1,11 +1,10 @@
 (ns cmr.message-queue.test.queue-broker-side-api
   "Defines routes for accessing a controlling the queue broker in testing through a side api and
   function for accessing those routes."
-  (:require [compojure.core :refer :all]
+  (:require [compojure.core :refer  [GET POST context routes]]
```

Comment:
Normally the CMR style guide wants the [compojure.core ... to go under the : after (:require